setup.py

```diff
@@ -7,7 +7,7 @@
 
 INSTALL_REQUIRES = [
     'django-cms>=3.4.5',
-    'Django>=1.11,<2.2',
+    'Django>=1.11,<2.3',
```

Comment:
There will be no 2.3, this should be <3.0.  https://www.djangoproject.com/weblog/2015/jun/25/roadmap/